### PR TITLE
[CST-2180] Add support form stats page

### PR DESCRIPTION
**Pull request by Lockyy**
### Context

- Ticket: https://dfedigital.atlassian.net/browse/CST-2180

### Changes proposed in this pull request

Add stats page for support forms

### CODE REVIEW

File: app/views/admin/performance/support_queries/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<%= render partial: "admin/performance/nav" %>
```
Comment (frankieroberto) [inline app/views/admin/performance/support_queries/show.html.erb:1]: We don't currently consistent set the page title for all the admin pages, but let's try and do it for all the new ones... 😂 🤦 

```suggestion
<% content_for :title, "Support form stats" %>

<%= render partial: "admin/performance/nav" %>
```

File: app/views/admin/performance/support_queries/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+          <% row.with_cell(header: true, text: "Number of users", classes: "govuk-!-text-align-right") %>
+        <% end %>
+      <% end %>
+
+      <% table.with_body do |body| %>
+        <% @subjects.each do |support_query_subject| %>
+          <% body.with_row(html_attributes: { data: { subject: support_query_subject }}) do |row| %>
+            <% row.with_cell(text: t("support_query.stats.#{support_query_subject}")) %>
+            <% row.with_cell(html_attributes: { data: { stat: :total } }, text: @support_query_counts[support_query_subject] || 0, numeric: true) %>
+            <% row.with_cell(html_attributes: { data: { stat: :users } }, text: @support_query_user_counts[support_query_subject] || 0, numeric: true) %>
```
Comment (frankieroberto) [inline app/views/admin/performance/support_queries/show.html.erb:22]: Not critical given our current low numbers, but usually worth using `number_with_delimiter` or similar to add comma separators for the 1,000s - just to make the numbers more readable?

```suggestion
            <% row.with_cell(html_attributes: { data: { stat: :total } }, text: number_with_delimiter(@support_query_counts[support_query_subject] || 0), numeric: true) %>
            <% row.with_cell(html_attributes: { data: { stat: :users } }, text: number_with_delimiter(@support_query_user_counts[support_query_subject] || 0), numeric: true) %>
```

**Review by frankieroberto — APPROVED** [review]
Ran this locally and looks good to me!

Left some minor content change suggestions.

**Review by leoapost — APPROVED** [review]

File: app/views/admin/performance/support_queries/show.html.erb (hunk 2 above)
Comment (Lockyy) [reply app/views/admin/performance/support_queries/show.html.erb:24]: Absolutely, good catch

**Review by leoapost — APPROVED** [review]

**Pull request merged**